src/main.js

```diff
@@ -63,16 +63,24 @@ function message(msg) {
   }, 3000);
 }
 
+const prefetched = [];
 function prefetch(e) {
   if (e.target.tagName != "A") {
     return;
   }
   if (e.target.origin != location.origin) {
     return;
   }
+  if (window.location.pathname === e.target.pathname) {
+    return;
+  }
+  if (prefetched.includes(e.target.href)) {
```

Comment:
Please remove. The way the prefetch cache works is that it will cache the returning resource for 5 minutes independent of its caching headers and duplicate fetches should not happen. If this is not the case, please provide a DevTools screenshot showing it and ideally file a crbug.com